pkg/proxy/ipvs/proxier.go

```diff
@@ -739,6 +739,8 @@ func (proxier *Proxier) syncProxyRules() {
 	activeIPVSServices := map[string]bool{}
 	// currentIPVSServices represent IPVS services listed from the system
 	currentIPVSServices := make(map[string]*utilipvs.VirtualServer)
+	// activeBindAddrs represents ip address successfully bind to DefaultDummyDevice in this round of sync
+	activeBindAddrs := map[string]bool{}
```

Comment:
Why don't leverage sets.String?

Reply:
Because we need a map (activeBindAddrs) to recode these addresse which successfully bind to DefaultDummyDevice. To check whether an element in `currentBindAddrs` is active, map is more effective (O(1)) than string set. Just like what `activeIPVSServices` and `currentIPVSServices` do